R/measures.R

```diff
@@ -0,0 +1,27 @@
+#' @title Measure Registry
+#'
+#' @description
+#' The [environment()] `measures` keeps track of all measures in this package.
+#' It stores meta information such as minimum, maximum or if the
+#' measure must be minimized or maximized.
+#'
+#' @export
+#' @examples
+#' names(measures)
+#' measures$tpr
+measures = new.env(parent = emptyenv())
+
+# adds items to registry
+add_measure = function(obj, title, type, lower, upper, minimize) {
```

Comment:
So the measures registry is `mlr_measures` and this is available if you import `mlr_measures`.
You can simply add to this registry using `mlr_measures$add`. 
See https://github.com/mlr-org/mlr3proba/blob/7f7dc61a29d7fd91849027eb971e7e0765cfb0c7/R/zzz.R#L123 for an example on how proba adds to mlr_pipeops

Reply:
This means, that this whole file is most likely not needed.

Reply:
Deleted the profile, sorry i just forgot to delete it.